<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuotationPage</class>
 <widget class="QWidget" name="QuotationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="sortPriceBtn">
   <property name="geometry">
    <rect>
     <x>308</x>
     <y>24</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Current Price</string>
   </property>
   <property name="iconSize">
    <size>
     <width>6</width>
     <height>10</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="sortChangeRateBtn">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>24</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Profit/Loss</string>
   </property>
   <property name="iconSize">
    <size>
     <width>6</width>
     <height>10</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="sortChangeBtn">
   <property name="geometry">
    <rect>
     <x>658</x>
     <y>24</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Growth/Decline</string>
   </property>
   <property name="iconSize">
    <size>
     <width>6</width>
     <height>10</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="nameBtn">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>24</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Assets</string>
   </property>
   <property name="iconSize">
    <size>
     <width>6</width>
     <height>10</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
